server/controllers/auth/login.js

```diff
@@ -0,0 +1,85 @@
+// modules
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const database = require("../../database");
+
+exports.login = (req, res, next) => {
+  // todo: 'remember me' feature
+  const email = req.body.email;
+  const password = req.body.password;
```

Comment:
You could de-structure them whenever possible.

Reply:
Also, you could implement Joi validation here by taking reference from my PR? I think it has same exact properties.

Reply:
> You could de-structure them whenever possible.

What do you mean by de-structure?

> Also, you could implement Joi validation here by taking reference from my PR? I think it has same exact properties.

Will do that once this PR is merged.

Reply:
` const { email, password } = req.body;` 
something like that.

Reply:
Keeping it expanded makes the code readable.